source/common/tracing/zipkin/zipkin_tracer_impl.cc

```diff
@@ -112,19 +112,11 @@ Tracing::SpanPtr Driver::startSpan(const Tracing::Config& config, Http::HeaderMa
     uint64_t traceId(0);
```

Comment:
Sorry didn't catch this before. Please use snake_case for variables (trace_id, span_id, etc.).